sonar-core/src/main/java/org/sonar/core/config/CorePropertyDefinitions.java

```diff
@@ -282,6 +285,14 @@ private CorePropertyDefinitions() {
         .category(CoreProperties.CATEGORY_EXCLUSIONS)
         .subCategory(CoreProperties.SUBCATEGORY_DUPLICATIONS_EXCLUSIONS)
         .multiValues(true)
+        .build(),
+
+      // ORGANIZATIONS
+      PropertyDefinition.builder(ORGANIZATIONS_ANYONE_CAN_CREATE)
+        .defaultValue(Boolean.toString(ORGANIZATIONS_ANYONE_CAN_CREATE_DEFAULT_VALUE))
```

Comment:
for readability I would recommend to put default value after the name. Moreover why using a constant ? For the same readability reason, it's better to have a full understanding of property when reading this block.